### PR TITLE
CAD-1264: bump `cardano-node`

### PR DESCRIPTION
**Pull request by deepfire**
1. bump `cardano-node`
1. update `scripts/analyse.sh` to log format changes

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -15,45 +15,31 @@ packages:
...
-  tag: f8a2da1edc2d1c5389c41d069ae9e2d8e848958a
-  --sha256: 05nv40b6p1mwfvfxpas7b5d470dniym4vpslnxqpj3i0w7qcicpb
+  tag: 9cfe6fc4236ca38cfa7245ffd70feb161badd79a
+  --sha256: 03v7p2hap2wfdwdf7sflwc78cqqg5qn3d9rww0xq982rijdxvvsx
   subdir: cardano-node
 
-source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/cardano-db-sync
-  tag: 538fce2c9cb711e853cf15f22f37f55a73e4f1b4
```
Comment (CodiePP) [inline cabal.project]: please, just comment these out. We might need to integrate it again in the future.

**Review by CodiePP — CHANGES_REQUESTED** [review]
"fixup" into a single commit, please.

File: cabal.project (hunk 1 above)
Comment (deepfire) [reply cabal.project:46]: This is only needed if we want `cardano-db-sync` as a library.

We still have it in `sources.json` for the executables.

I don't suppose we ever used the proxy as a library?

**Comment by deepfire** [comment]
Done.

**Review by CodiePP — APPROVED** [review]

**Pull request merged**